content/en/docs/Advanced/sys-extensions.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Extending the system with system extensions"
+linkTitle: "Extending the system with system extensions"
```

Comment:
Otherwise it sounds generic, as if it would work on all flavors.

```suggestion
title: "Extending the system with systemd extensions"
linkTitle: "Extending the system with systemd extensions"
```

Reply:
well the problem is that the name is `system extension images` so it would be `systemd system extensions` which is terrible xDDD